src/parsers/manifestjson.js

```diff
@@ -510,8 +510,12 @@ export default class ManifestJSONParser extends JSONParser {
     try {
       const info = await getImageMetadata(this.io, iconPath);
       if (info.width !== info.height) {
-        this.collector.addError(messages.iconIsNotSquare(iconPath));
-        this.isValid = false;
+        if (info.mime !== 'image/svg+xml') {
+          this.collector.addError(messages.iconIsNotSquare(iconPath));
+          this.isValid = false;
+        } else {
+          this.collector.addWarning(messages.iconIsNotSquare(iconPath));
+        }
```

Comment:
:+1: this change on the implementation side looks the one I was thinking of.

This PR should only be missing a new test case now, would you mind to define a new test case to cover the warning on non square svg icons?

you could start by looking to the test case that you were changing in the previous version of this PR, and based on that create a new test case to cover the new expected behavior.

Reply:
Sure, i have already made one, i'll commit it right now, please correct me if anything is wrong. Thank you.